### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by Abduganeoff** [DevSkillsHQ/discusssions#36] [Help] [answered]
Hi, I am wondering should I use this API to send API requests from backend https://infra.devskills.app/accounting/api/3.1.0 or I should/can use a server e.g. MongoDB?

Please let me know at your earliest convenience.

Regards,
M

**Comment by fenske** [comment, marked as answer]
Hi @Abduganeoff. I'm Anton from Alva. The expectation is for you to build a backend that implements the referenced API spec. You're also advised to keep data in an in-memory data store to keep it simple. 

Hope this clarifies it.

Let me know if you have any other questions.

**Comment by hasanamy** [comment]
very helpfulll







https://steam-shop.ir/product/%D8%AE%D8%B1%DB%8C%D8%AF-nord-vpn-%D8%A7%DB%8C%D9%BE%DB%8C-%D8%AB%D8%A7%D8%A8%D8%AA/
